samples/README.md

```diff
@@ -0,0 +1,116 @@
+# Azure Smoke Tests for Python
```

Comment:
CI tests failed because we need to update the regex in `/.docsettings.yml`. 

The errors were: 
```
 * Azure .+ client library for Python
 * ^Key concepts$
```

To fix these: 
1. update the expression in `.docsettings.yml` to include the smoke test heading
2. add a "Key concepts" section